src/modules/products/services/product.service.ts

```
@@ -0,0 +1,77 @@
+    try {
+      const product = this.productRepository.create(createProductDTO);
+      return await this.productRepository.save(product);
+    } catch (error) {
+      throw new BadRequestException('Failed to create product');
+    }
+  }
+
+  async findAll(): Promise<Product[]> {
+    try {
+      return await this.productRepository.find();
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)